lib/src/google_sign_in_mocks_base.dart

```diff
@@ -4,13 +4,22 @@ import 'package:mockito/mockito.dart';
 class MockGoogleSignIn extends Mock implements GoogleSignIn {
   MockGoogleSignInAccount _currentUser;
 
+  /// To simulate google login cancellation behaviour.
+  bool _isCancelled = false;
+
+
+  void setIsCancelled(bool val) {
+    _isCancelled = val;
+  }
```

Comment:
Can you add an empty line after the method?

Reply:
Done.